src/org/jitsi/impl/neomedia/rtp/RTPEncodingDesc.java

```diff
@@ -226,6 +226,7 @@ else if (tsDiff >= (-(1L << 30) & 0xFFFFFFFFL))
                 }
                 else if (snDiff < 2 || snDiff > (-3 & 0xFFFF))
                 {
+                    // FIXME we get that way too often.
```

Comment:
We should probably have `snDiff = RTPUtils.sequenceNumberDiff(max, min);` but I'll fix that in another PR as it doesn't break anything (source frame identification is for temporal scalability which is not pushed).